src/utils/checkImageNsfw.ts

```diff
@@ -0,0 +1,37 @@
+import { ImageFlagging } from '../models/imageFlagging';
+
+interface Options {
+
+  /** Whether image is nsfw. */
+  isImageNsfw?: boolean | null;
+
+  /** Image flagging. */
+  imageFlagging?: ImageFlagging | null;
+}
+
+// Sexual content rated higher than 0.1 can contain massive boobas.
```

Comment:
```suggestion
// Sexual content rated higher than 0.1 may contain nudity.
```

Reply:
There probably won't be nudity under 1, but massive boobas that may take >50% of a poster.